composer.json

```diff
@@ -15,8 +15,8 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.3 || ^8",
-        "phpspec/prophecy": "^1.3",
-        "phpunit/phpunit":"^9.1"
+        "phpspec/prophecy": "^1.18 || dev-master as 1.18",
+        "phpunit/phpunit":"^9.1 || ^10.1"
```

Comment:
This is probably not taggable as-is, because without `--prefer-stable` or `--prefer-dist`, it would probably install an unstable version of Prophecy. As soon as https://github.com/phpspec/prophecy/pull/585 is tagged we can pin against that though.

Reply:
With d613325 this is no longer true, and it already works in CI! Great!